x-pack/plugins/apm/server/lib/correlations/latency/get_latency_distribution.ts

```diff
@@ -124,20 +115,14 @@ export async function getLatencyDistribution({
       }));
     }
 
-    return {
-      distributionInterval,
-      overall: {
-        distribution: formatDistribution(response.aggregations.distribution),
-      },
-      significantTerms: topSigTerms.map((topSig, index) => {
-        // @ts-expect-error
-        const agg = response.aggregations[`term_${index}`] as Agg;
+    return topSigTerms.map((topSig, index) => {
+      // @ts-expect-error
```

Comment:
Why is it necessary?

Reply:
I added a line comment with a brief explanation